app/src/components/api.ts

```diff
@@ -0,0 +1,12 @@
+export const PRODUCTION = process.env.NODE_ENV === "production";
+
+export const API_BASE_URL =
+  process.env.REACT_APP_API_BASE_URL || "https://api.speedcubingcanada.org";
+
+export const signIn = () => {
+  window.location.assign(new URL("/login", API_BASE_URL));
+};
+
+export const signOut = () => {
+  window.location.assign(new URL("/logout", API_BASE_URL));
+};
```

Comment:
Is it possible to use react navigation to get to login/logout? I think that would be better practice than editing the window location directly.

Reply:
That's what I tried first but for some reason it wasn't working. I agree with you that it would be more coherent to have it like the rest and I would be happy to change it if we find a way that works.

Reply:
I don't recall at the moment how to change this. Maybe Jon can have a quick look when he has time.